android/sdk/src/main/java/com/taobao/weex/utils/WXSoInstallMgrSdk.java

```diff
@@ -381,18 +351,19 @@ private static String _getFieldReflectively(Build build, String fieldName) {
   }
 
   private static String _cpuType() {
-    String abi ;
-    try {
-      abi = Build.CPU_ABI;
-    }catch (Exception e){
-      e.printStackTrace();
-      abi = ARMEABI;
-    }
-    if (TextUtils.isEmpty(abi)){
-      abi = ARMEABI;
+    if(TextUtils.isEmpty(mAbi)) {
+      try {
```

Comment:
1. No need to add try catch. Even if there is `NoSuchFieldError`, it is an error, not an exception, so `catch Exception` is useless.
1. Use `Build.SUPPORTED_ABIS` as `Build.CPU_ABI` is deprecated.